src/Donations/Components/PaymentStatus.tsx

```diff
@@ -122,52 +124,56 @@ function ThankYou() {
 
   return (
     <div className="donations-forms-container" style={{ paddingBottom: "0px" }}>
-      <div className="donations-form w-100">
-        {!ready && !donation ? (
-          <PaymentProgress isPaymentProcessing={true} />
-        ) : (
-          <div>
-            {donation && donation.paymentStatus ? (
-              status === "success" ||
-              status === "paid" ||
-              status === "succeeded" ? (
-                <SuccessComponent />
-              ) : status === "failed" || paymentError ? (
-                <FailedDonation
-                  donationID={donationID}
-                  sendToReturn={sendToReturn}
-                  donation={donation}
-                />
-              ) : transferDetails ? (
-                <TransferDetails
-                  donationID={donationID}
-                  donation={donation}
-                  sendToReturn={sendToReturn}
-                />
+      {projectDetails?.purpose === "planet-cash-signup" ? (
```

Comment:
@Shreyaschorge While functionality works fine, I do feel that we should keep a separate donation step for the planet cash signup form, and not include it within PaymentStatus. Perhaps this can be kept as a future refactor.

Reply:
This is still under discussion.
There is an open issue https://github.com/Plant-for-the-Planet-org/planet-donations/issues/277 in which donation step needs to be removed from the project.
This needs to be done in a different PR.

Reply:
I've made a note of it in #277.